src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
```

Comment:
the rightmost star don't need a `marign` 🥲

<img width="253" alt="image" src="https://user-images.githubusercontent.com/55802630/172805789-8ec7f2ce-72c7-4598-8a5b-5b6f1998ab02.png">